docs/instruments/LORISLogicParser.md

```diff
@@ -0,0 +1,143 @@
+
+### If Logic
+| Operation 	| Syntax       	| Notes                                                                            	|
+|-----------	|--------------	|----------------------------------------------------------------------------------	|
+| if        	| if(cond,x,y) 	| If the condition 'cond' evaluates to true, x is returned. If not, y is returned. 	|
+
+### Functions
+| Operation          	| Syntax               	| Notes                                                                                	|
+|--------------------	|----------------------	|--------------------------------------------------------------------------------------	|
+| not a number       	| isNaN(a)             	| returns true if a is not a number                                                    	|
+| modulo             	| mod(a, b)            	| returns the remainder of a / b                                                       	|
+| round              	| round(a, b)          	| rounds a to b decimal places; note all rounding functions support trailing 0s        	|
+| round up           	| roundup(a, b)        	| rounds a up to b decimal places                                                      	|
+| round down         	| rounddown(a,b)       	| rounds a down to b decimal places                                                    	|
+| square root        	| sqrt(a)              	|                                                                                      	|
+| absolute value     	| abs(a)               	| returns the positive value of a                                                      	|
+| minimum            	| min(a,b,c,d...)      	| returns the smallest value of its arguments                                          	|
+| maximum            	| max(a,b,c,d...)      	| returns the largest value of its arguments                                           	|
+| mean               	| mean(a,b,c,d...)     	| returns the average value of its arguments                                           	|
+| median             	| median(a,b,c,d...)   	| sorts its arguments in ascending order and returns the median                        	|
+| sum                	| sum(a,b,c,d...)      	| returns the sum of its arguments                                                     	|
+| product            	| product(a,b,c,d...)  	| returns the product of its arguments                                                 	|
+| variance           	| variance(a,b,c,d...) 	| returns the population variance of its arguments                                     	|
+| standard deviation 	| stdev(a,b,c,d...)    	| returns the population standard deviation (square root of variance) of its arguments 	|
+
+### Date Operations
+| Operation       	| Syntax                                        	|
+|-----------------	|-----------------------------------------------	|
+| date difference 	| datediff(date1, date2, units, signed)         	|
```

Comment:
what type is the return value of a date difference?

Reply:
in JS, a number. in PHP, a float. essentially both a decimal